demo/index.html

```diff
@@ -1,17 +1,87 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <style>
-    .text {
-      padding: 20px;
-    }
-  </style>
   <link rel="stylesheet" href="../.tmp/annotator-content-analysis.css" type="text/css">
   <link rel="stylesheet" href="../vendor/bower_components/bootstrap/dist/css/bootstrap.css" type="text/css">
+  <!-- temp -->
+  <link rel="stylesheet" href="../.tmp/facelift.css" type="text/css">
+  <link href='https://fonts.googleapis.com/css?family=PT+Serif' rel='stylesheet' type='text/css'>
```

Comment:
Hm, not experiencing any mixed content issues on my end. Isn't that usually for pages served through https including plain http content?